docs/src/example/drawerNavItem/BasicDrawerNavItemExample.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { Box } from '@mui/material';
+import { Drawer, DrawerBody, DrawerNavGroup, DrawerNavItem } from '@brightlayer-ui/react-components';
+import * as colors from '@brightlayer-ui/colors';
+
+export const BasicDrawerNavItemExample = (): JSX.Element => {
+    return (
+        <Box sx={{ m: '16px 0', backgroundColor: colors.white[600], p: 4 }}>
```

Comment:
You can make use of the theme props when using sx.

```suggestion
        <Box sx={{ m: 2 0, backgroundColor: colors.white[600], p: 4 }}>
```